lib/Driver.cpp

```diff
@@ -203,10 +218,10 @@ void Driver::debug_PrintMutationPoints() {
 
   for (auto &Test : Finder.findTests(Ctx)) {
     outs() << Test->getTestName() << "\n";
-    for (auto testee : Finder.findTestees(Test.get(), Ctx, Cfg.getMaxDistance())) {
-      auto MPoints = Finder.findMutationPoints(Ctx, *(testee.first));
+    for (auto &&testee : Finder.findTestees(Test.get(), Ctx, Cfg.getMaxDistance())) {
```

Comment:
I don't think you really need the universal reference here. I think it's safer to take the lvalue reference and "move" it if you want to transfer ownership. Now it's hard to tell if ownership it transferred in the first place and where that happens.